src/Bogardo/Mailgun/Mailgun.php

```diff
@@ -98,7 +98,7 @@ public function send($view, array $data, $callback, $mustInit = true)
 
 		$this->getMessage($view, $data);
 
-		return $this->mailgun()->sendMessage(Config::get('mailgun::domain'), $this->getMessageData(), $this->getAttachmentData());
+		return $this->mailgun(true, Config::get('mailgun::api_key'))->sendMessage(Config::get('mailgun::domain'), $this->getMessageData(), $this->getAttachmentData());
```

Comment:
No. It was automatically picked when I selected the Bogardo:develop branch. Thanks